packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1072,6 +1072,11 @@
       <shortDescription>Inclusion filter on field; no results</shortDescription>
     </labels>
     <labels>
+      <fullName>quantic_OpensInSalesforceSubTab</fullName>
+      <value>Opens in a salesforce console subTab</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Opens in a salesforce console subTab</shortDescription>
```

Comment:
the labels tag is not closed properly, please confirm that labels are properly closed in this file.
```suggestion
      <shortDescription>Opens in a salesforce console subTab</shortDescription>
    </labels>
    <labels>
```